apps/meteor/client/sidebar/footer/voip/VoipFooter.tsx

```diff
@@ -104,8 +114,15 @@ export const VoipFooter = ({
 								color={paused ? 'neutral-500' : 'info'}
 								small
 								square
+								secondary
 								onClick={handleHold}
 							/>
+							{paused ? (
+								<Icon name='pause-unfilled' color='neutral-500' size='x24' />
+							) : (
+								<Icon name='pause-unfilled' color='info' size='x24' />
+							)}
```

Comment:
You could use only one component:
`<Icon name='pause-unfilled' color={paused ? 'neutral-500' : 'info'} size='x24' />`